lib/plugins/inspec-plugin-manager-cli/lib/inspec-plugin-manager-cli/cli_command.rb

```diff
@@ -478,6 +478,16 @@ def make_pretty_version(status)
           "src"
         end
       end
+
+      def make_pretty_install_type(status)
+        {
+          bundle: "bundle", # We could call this core, too - not much of a distinction
+          core: "core",
+          path: "path",
+          gem: "gem (user)",
+          system: "gem (system)",
+        }[status.installation_type]
```

Comment:
Should extract the static hash out to a frozen constant (unless you anticipate it going dynamic later (tho that might be a merge of the constant + dynamic))